frontend/javascripts/messages.tsx

```diff
@@ -254,10 +254,10 @@ instead. Only enable this option if you understand its effect. All layers will n
     'All trees are currently hidden. You can disable this by toggling the "Skeleton" layer in the layer settings in the left sidebar.',
   "tracing.invalid_json_url_hash":
     "Cannot parse JSON URL hash. More information was printed to the browser's console.",
-  "tracing.pin_mapping_info":
-    "The active volume annotation layer has an active mapping. By mutating the layer, the mapping will be permanently enabled. This can only be undone by restoring an older version of this annotation. Are you sure you want to continue?",
-  "tracing.pin_mapping_confirmed": (mappingName: string) =>
-    `The mapping ${mappingName} is now permanently enabled for this annotation and can no longer be changed or disabled.`,
+  "tracing.locked_mapping_info":
+    "The active volume annotation layer has an active mapping. By mutating the layer, the mapping will be permanently locked (and can no longer be changed or disabled). This can only be undone by restoring an older version of this annotation. Are you sure you want to continue?",
```

Comment:
I wanted the parentheses to indicate that this part would be optional. I think it's more clear to include it, but would remove the parentheses, then :)

```suggestion
    "The active volume annotation layer has an active mapping. By mutating the layer, the mapping will be permanently locked and can no longer be changed or disabled. This can only be undone by restoring an older version of this annotation. Are you sure you want to continue?",
```